src/pulselistener/pulselistener/lib/bus.py

```diff
@@ -35,8 +59,15 @@ def add_queue(self, name, mp=False, maxsize=-1):
         '''
         assert name in self.queues, 'Missing queue {}'.format(name)
         queue = self.queues[name]
-        if isinstance(queue, asyncio.Queue):
+
+        if isinstance(queue, RedisQueue):
+            async with AsyncRedis() as redis:
```

Comment:
Maybe it'd be better to connect to redis only once instead of each time we send a message

Reply:
Ok, will do.

Reply:
Done.